user-interface/src/case-detail/CaseDetailScreen.tsx

```diff
@@ -222,6 +228,23 @@ export default function CaseDetail(props: CaseDetailProps) {
       });
   }
 
+  async function fetchCaseAuditHistory() {
+    setIsAuditHistoryLoading(true);
+    api
+      .get(`/cases/${caseId}/history`, {})
+      .then((data) => {
+        const response = data as CaseAuditHistoryResponseData;
+        if (response) {
+          setCaseAuditHistory(response.body);
+          setIsAuditHistoryLoading(false);
+        }
+      })
+      .catch(() => {
+        setCaseAuditHistory([]);
+        setIsAuditHistoryLoading(false);
+      });
```

Comment:
I see we're catching this but we aren't providing any notice to the user that there was an issue... if there was.

Reply:
Two cases: 404 and 500.